cli/impl/src/main/java/io/helidon/build/cli/impl/InitOptions.java

```diff
@@ -107,6 +108,8 @@ public String toString() {
             @KeyValue(name = "artifactid", description = "Project's artifact ID") String artifactId,
             @KeyValue(name = "package", description = "Project's package name") String packageName,
             @KeyValue(name = "name", description = "Project's name") String projectName,
+            @KeyValue(name = "output-props-file", description = "Path to file where user inputs will be saved")
+            String outputPropsFile,
```

Comment:
Can this be a `Path` directly ?

Reply:
Supported value types are `String.class, Integer.class, File.class, Enum.class`